neo4j-store/src/main/kotlin/org/ostelco/prime/storage/graph/Neo4jStore.kt

```diff
@@ -747,7 +747,8 @@ object Neo4jStoreSingleton : GraphStore {
     override fun provisionSimProfile(
             identity: ModelIdentity,
             regionCode: String,
-            profileType: String?): Either<StoreError, ModelSimProfile> = writeTransaction {
+            profileType: String?,
+            alias: String): Either<StoreError, ModelSimProfile> = writeTransaction {
         IO {
```

Comment:
High indentation, complex control flow.   I find this hard to read. Please simplify.

Reply:
@la3lma There is an ongoing effort to improve and reduce the size of the `Neo4jStore` code. Part of @vihangpatil "rule engine" plans.